src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -18,16 +20,33 @@ const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
     firstEnrollablePaidSeatPrice,
     aggregationKey,
   } = original;
-
+  const trackEvent = () => {
+    const trackInfo = {
+      content_metadata: {
+        aggregation_key: aggregationKey,
+      },
+    };
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_CONFIRM_CONTENT_ABOUT_PAGE}.clicked`,
```

Comment:
nit: typically the `.clicked` portion of the event name should be included in the constant itself, too.